src/Compilers/CSharp/Test/Syntax/Syntax/SyntaxNormalizerTests.cs

```diff
@@ -16,6 +16,42 @@ namespace Microsoft.CodeAnalysis.CSharp.UnitTests
 {
     public class SyntaxNormalizerTests
     {
+        [Fact]
+        [WorkItem(50742, "https://github.com/dotnet/roslyn/issues/50742")]
+        public void TestLineBreakInterpolations()
+        {
+            var code = @"$""Printed: { new Printer() { TextToPrint = ""Hello world!"" }.PrintedText }""";
+
+            var syntaxNode = SyntaxFactory.ParseExpression(code).NormalizeWhitespace();
+            var text = syntaxNode.ToFullString();
+
+            //we test only for line breaks because they caused that the produced text couldn't be parsed after NormalizeWhitespace
+            //the concrete formatting (e.g. spaces, tabs) should be ignored here
+            Assert.True(!text.Contains("\n"));
+
+            var parsed = SyntaxFactory.ParseExpression(text);
+            Assert.False(parsed.HasErrors);
```

Comment:
please add an assert about what the expected value is as well.

Reply:
you can use the TestNormalizeExpression helper.

Reply:
I don't want to define the actual formatting - I don't care. I just want that the re-parsed text has no errors.

Reply:
Should the test fail, just because initializer is printed with or without newline? Or if the indention is changed? In my opinion not. Especially this test is not really testing the formatting - it's just checking if formatting does not destroy the syntax.

Reply:
I can add it, but I don't like it. I would even remove the "\n" check. If it would compile it would be fine for me!

Reply:
> I don't want to define the actual formatting 

I get that. But I do :-)

Reply:
> I can add it, but I don't like it. 

Please add it :-)

Reply:
I did it :-)